demo/demo.py

```diff
@@ -0,0 +1,11 @@
+import pandas as pd
+
+data = [['a','b','c'], ['1','2','3','4']]
+
+def createDataFrame():
+  df = pd.DataFrame(data, columns=['name', 'value', 'key', 'values'])
```

Comment:
**Suggestion:** Use a more descriptive variable name for the DataFrame in the createDataFrame() function.
```suggestion
  +  dataframe = pd.DataFrame(data, columns=['name', 'value', 'key', 'values'])
```